anago

```diff
@@ -203,25 +203,29 @@ check_prerequisites () {
        (logrun gcloud config set account $user && \
         logrun gcloud docker -- version >/dev/null 2>&1); then
       logecho -r "$OK"
+      break
```

Comment:
I'm confused as to why you're breaking out of the loop here. Won't this result in only checking `$G_AUTH_USER`?